code/game/gamemodes/clock_cult/clock_scriptures/scripture_drivers.dm

```diff
@@ -98,6 +98,49 @@
 	Click your slab to cancel.</b></span>"
 
 
+//Abscond: Used to return to Reebe.
+/datum/clockwork_scripture/abscond
+	descname = "Return to Reebe"
+	name = "Abscond"
+	desc = "Yanks you through space, returning you to home base."
+	invocations = list("As we bid farewell, and return to the stars...", "...we shall find our way home.")
+	whispered = TRUE
+	channel_time = 50
+	usage_tip = "This can't be used while on Reebe, for obvious reasons."
+	tier = SCRIPTURE_DRIVER
```

Comment:
I feel like spatial gateway is quite powerful in its own right, and I think a basic "hey get out of here" scripture is fine in a lot of cases.